gulpfile.babel.js

```diff
@@ -32,6 +33,7 @@ const resources = [
   'node_modules/typescript/bin/*',
   'node_modules/livescript/bin/*',
   'node_modules/font-manager/**/*',
+  'node_modules/font-awesome/**/*'
```

Comment:
@dstreet font-awesome is captured from `npm list --parse --prod` command.
![image](https://cloud.githubusercontent.com/assets/1622515/13090466/f7616f5a-d51b-11e5-8e3d-23416c808b62.png)